app/views/vm_common/console_vmrc.html.haml

```diff
@@ -367,36 +367,42 @@
         // Start up VMRC
         vmrc = document.getElementById("vmrc");
 
-        vmNameTooltip = $("#vmNameTooltip");
-        var vmTitle = $("#vmTitle");
+        if (typeof vmrc.isReadyToStart === 'undefined') {
+          window.location.href = "vmrc://clone:#{ticket}@#{host}:443/?moid=#{vmid}";
```

Comment:
> is it enough to wrap the host variable around []?

Only with ipv6.  With a regular hostname or ipv4 address you *don't* do that.  Is there a library that handles URl building for you that Does The Right Thing™ ?